### PR TITLE
docs(tutorials/jokes): remove unused imports

### PR DESCRIPTION
**Pull request by arifqys**
Hello👋 I made small update on the docs by removing some imports that are not used anywhere in that file.

### CODE REVIEW

File: docs/tutorials/jokes.md [hunk 1]
```diff
@@ -1802,7 +1802,6 @@ const [randomJoke] = await db.joke.findMany({
 <summary>app/routes/jokes/index.tsx</summary>
 
 ```tsx filename=app/routes/jokes/index.tsx lines=[5,7-15,18]
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:1804]:
```suggestion
```tsx filename=app/routes/jokes/index.tsx lines=[1-2,4,6-14,17]
```

File: docs/tutorials/jokes.md [hunk 2]
```diff
@@ -2523,7 +2522,6 @@ import type {
   ActionArgs,
   LinksFunction,
 } from "@remix-run/node";
-import { json } from "@remix-run/node";
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:2526]: Be sure to fix code highlighting as well

File: docs/tutorials/jokes.md [hunk 3]
```diff
@@ -3692,7 +3687,6 @@ import type {
...
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:3695]: Same

File: docs/tutorials/jokes.md [hunk 4]
```diff
@@ -5017,7 +5011,6 @@ import type {
   LinksFunction,
   MetaFunction,
 } from "@remix-run/node";
-import { json } from "@remix-run/node";
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:5020]: Same

File: docs/tutorials/jokes.md [hunk 5]
```diff
@@ -5755,7 +5748,6 @@ import type {
 } from "@remix-run/node";
 import { json, redirect } from "@remix-run/node";
 import {
-  Link,
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:5758]: Same

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: docs/tutorials/jokes.md (hunk 2 above)
Comment (arifqys) [reply docs/tutorials/jokes.md:2526]: Thanks for the review, updated @MichaelDeBoey

File: docs/tutorials/jokes.md (hunk 3 above)
Comment (arifqys) [reply docs/tutorials/jokes.md:3695]: Updated

File: docs/tutorials/jokes.md (hunk 4 above)
Comment (arifqys) [reply docs/tutorials/jokes.md:5020]: Updated

File: docs/tutorials/jokes.md (hunk 5 above)
Comment (arifqys) [reply docs/tutorials/jokes.md:5758]: Updated

**Review by MichaelDeBoey — APPROVED** [review]

**Pull request merged**